tests/baselines/reference/privateNameDuplicateField.errors.txt

```diff
@@ -0,0 +1,13 @@
+tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts(5,5): error TS2300: Duplicate identifier '#foo'.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNameDuplicateField.ts (1 errors) ====
+    // @target es6
+    
+    class A {
+        #foo = "foo";
+        #foo = "foo";

```

Comment:
should have circular-reference and use-before-def tests, too. (eg `#foo = 1 + #foo` and `#foo = #bar; #bar = #foo`) 
There might some below. I'll delete this comment if so.